pkg/webhooks/kafkacluster_validator_test.go

```diff
@@ -486,3 +487,221 @@ func TestCheckBrokerStorageRemoval(t *testing.T) {
 		}
 	}
 }
+
+func TestCheckExternalListenerStartingPort(t *testing.T) {
+	testCases := []struct {
+		testName         string
+		kafkaClusterSpec v1beta1.KafkaClusterSpec
+		isValid          bool
+	}{
+		{
+			// In this test case, all resulting external port numbers should be valid
+			testName: "valid-2-brokers-2-externalListeners",
```

Comment:
Question: is there a particular reason to add the dashes in the `testName`? If not, removing the dashes would make it a bit more readable IMO

```suggestion
			testName: "valid configurations: 3 brokers with 2 externalListeners",
```

Reply:
The reason was to keep it the same in the `go test` output (subtests replace white spaces with `_`) as in code but that's not important. It is probably better to use the spaces. I'll update it.